### PR TITLE
Charlon/photo class list

### PR DESCRIPTION
**Pull request by charlon**
Styles list and grid view class list photos to be printable.

### CODE REVIEW

File: myuw/static/css/photo_list.less [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        li { border: solid 1px #ddd; margin-bottom: 16px; padding: 8px;
+
+            &:after { visibility: hidden; display: block; content: ""; clear: both; height: 0; }
+
+            @media @tablet { border: none; }
+            @media @desktop { border: none; }
+
+            .myuw-photo-list-image { float: left; margin-right: 16px; }
+            .myuw-photo-list-content { float: left; margin-top: 8px; }
+            .myuw-photo-list-meta { border-top: solid 1px #eee; margin-top: 4px; padding-top: 4px; font-size: 13px; }
```
Comment (klpittman) [inline myuw/static/css/photo_list.less]: Didn't want to stick with the 8px grid here?

File: myuw/templates/handlebars/teaching/photo_list.html [hunk 2]
```diff
@@ -2,103 +2,216 @@
     {% tplhandlebars "photo_class_list" %}
 
 <div class="container">
-    <div class="row">
-        <div id="photoclasslist_popup">
+
+        <div id="photoclasslist_popup" class="myuw-photo-list">
+
             <div class="card"  data-name="PhotoClasslistPopupCard" data-type="card">
```
Comment (klpittman) [inline myuw/templates/handlebars/teaching/photo_list.html]: Should this class be swapped to "`myuw-card`" class?

File: myuw/templates/handlebars/teaching/photo_list.html [hunk 3]
```diff
@@ -2,103 +2,216 @@
...
+
                 <h3 class="scheduleHeading">{{ toTitleCase quarter }} {{ year }} | {{ sections.0.curriculum_abbr }} {{ sections.0.course_number }} {{ sections.0.section_id }} | SLN {{ sections.0.sln }}</h3>
```
Comment (klpittman) [inline myuw/templates/handlebars/teaching/photo_list.html]: what about the `myuw-card-title` class here?

File: myuw/templates/handlebars/teaching/photo_list.html [hunk 4]
```diff
@@ -2,103 +2,216 @@
...
-                        </select>
 
+                <div id="classlist_controls" class="row myuw-photo-list-controls" style="padding-bottom: 40px;">
+
+                    <div class="col-sm-5">
+                        Display:&nbsp;&nbsp;<a id="list_view" href="#"><i class="fa fa-list" aria-hidden="true"></i> Detailed (List)</a> | <a id="grid_view" href="#"><i class="fa fa-th" aria-hidden="true"></i> Simple (Grid)</a>
                     </div>
-                    <div class="pull-right">
-                        <a href="#" id="download_class_list"><i class="fa fa-download"></i>&nbsp;Download class list (Excel)</a> | <a href="javascript:window.print()" class="classlist-print"><i class="fa fa-print"></i>&nbsp;print</a>
+                    <div class="col-sm-3">
```
Comment (klpittman) [inline myuw/templates/handlebars/teaching/photo_list.html]: myuw__photo_class_list

Wondering if we put these controls in the same col-sm-x div if they would align better? Maybe there's another way.

File: myuw/templates/handlebars/teaching/photo_list.html [hunk 5]
```diff
@@ -2,103 +2,216 @@
...
+                                        {{ /each }}
+                                        {{ /if }}
+                                    </div>
+                                </div>
+
+
+                            </li>
+                        {{ /each }}
+
+                        {{ #each sections.0.registrations }}
```
Comment (klpittman) [inline myuw/templates/handlebars/teaching/photo_list.html]: We need to remove these extra lists you used for testing.

File: myuw/templates/handlebars/teaching/photo_list.html [hunk 6]
```diff
@@ -2,103 +2,216 @@
...
-                        </div>
-                    {{ /each }}
-                </div> -->
-        
+
+
+                                <span class="sr-only">Email:</span> <a href="mailto:{{ email }}">{{ email }}</a> <br />
+
+                                    <div class="myuw-photo-list-meta">
+                                        Student Number: {{ student_number }} <br />
```
Comment (klpittman) [inline myuw/templates/handlebars/teaching/photo_list.html]: should we use two `&nbsp;`s here between label and data like we did on the profile? Not a big deal either way here.

**Comment by bexar** [comment]
I'm not able to see change in display. I see list but Simple (grid) view not displaying for me.

File: myuw/templates/handlebars/teaching/photo_list.html (hunk 4 above)
Comment (bexar) [inline myuw/templates/handlebars/teaching/photo_list.html]: Would 'View:' be a better label than "Display:" ?

File: myuw/templates/handlebars/teaching/photo_list.html (hunk 2 above)
Comment (charlon) [inline myuw/templates/handlebars/teaching/photo_list.html]: ok this is fixed now. probably a merging issue.

File: myuw/static/css/photo_list.less (hunk 1 above)
Comment (charlon) [inline myuw/static/css/photo_list.less]: switched it to 16px

File: myuw/templates/handlebars/teaching/photo_list.html (hunk 3 above)
Comment (charlon) [inline myuw/templates/handlebars/teaching/photo_list.html]: yes. probably a merging issue. it's fixed.

File: myuw/templates/handlebars/teaching/photo_list.html [hunk 7]
```diff
@@ -2,103 +2,85 @@
...
+                                <div class="myuw-photo-list-content">
+                                    <div>
+                                        <strong>{{ surname }}, {{ name }}</strong><br />
+                                        <span class="sr-only">Email:&nbsp;&nbsp;</span><a href="mailto:{{ email }}">{{ email }}</a>
+                                    </div>
+                                    <div class="myuw-photo-list-meta">
+                                        <div class="pull-left" style="width:50%; min-width: 272px;">
+                                        Student Number:&nbsp;&nbsp;{{ student_number }} <br />
+                                        UW NetID:&nbsp;&nbsp;{{ netid }} <br />
+                                        Major:&nbsp;&nbsp;{{ #each majors }} {{ full_name }} {{ /each }}<br />
```
Comment (klpittman) [inline myuw/templates/handlebars/teaching/photo_list.html]: The space in here `{{ #each majors }} {{ full_name }} {{ /each }}` before full_name is causing 3 spaces after "UW NetID:"

Comment (klpittman) [inline myuw/templates/handlebars/teaching/photo_list.html]: Everything else is looking good!

Comment (charlon) [inline myuw/templates/handlebars/teaching/photo_list.html]: ok pushed the fix.

**Pull request merged**